main/2024/CG-08-27.md

```diff
@@ -13,9 +13,10 @@ No registration is required for VC meetings. The meeting is open to CG members o
 ## Agenda items
 
 1. Opening
-2. Unified approach to stack switching (Sam Lindley & Francis McCabe)
-   1. Vote for phase 2 
 1. Proposals and discussions
+   1. Unified approach to stack switching (Sam Lindley & Francis McCabe)
```

Comment:
@fgmccabe @slindley I just noticed that you didn't request a time slot here. Is 45 minutes enough?

Reply:
Should be enough, yes.